encrypted-media-respec.html

```diff
@@ -305,7 +305,7 @@ <h2>Definitions</h2>
             Known keys are exposed via the <a def-id="keyStatuses"></a> attribute.
           </p>
 
-          <p>Keys are considered known even after they become unusable, such as due to <a def-id="expiration-time">expiration</a> or if they are removed but a <a def-id="record-of-license-destruction"></a> is available.
+          <p>Keys are considered known even after they become unusable, such as due to <a def-id="expiration-time">expiration</a> or if they are removed but a <a def-id="record-of-license-destruction"></a> or <a def-id="record-of-key-usage"></a> is available.
```

Comment:
"record-of-key-usage" seems to be rendering as a blank string in the preview.

I ran into issues with this in my own PR, too.  I would never have known this without my predecessor pointing it out to me, but there is a JS file where definitions need to be added.  For an example, see the changes I made in encrypted-media.js in https://github.com/w3c/encrypted-media/commit/9deeb1a99f30b3fd1752757e8f00c02777f8e052#diff-283a6070a66ef1d23494b31ebbb806e8

Reply:
thank you. i've updated encrypted-media.js